library/utils.nim

```diff
@@ -0,0 +1,18 @@
+import std/[json, options, strutils]
+import results
+import ../waku/waku_core/time
+
+proc getProtoInt64*(node: JsonNode, key: string): Option[int64] =
```

Comment:
I wonder if we should return a `Result` instead so that we can better control any possible error

Reply:
I changed it to a result now. Please do check latest commit and let me know your thoughts!